src/set_theory/zfc/basic.lean

```diff
@@ -677,6 +677,27 @@ theorem singleton_injective : function.injective (@singleton Set Set _) :=
 @[simp] theorem to_set_sUnion (x : Set.{u}) : (⋃₀ x).to_set = ⋃₀ (to_set '' x.to_set) :=
 by { ext, simp }
 
+/-- The intersection operator, the collection of elements in all of the elements of a ZFC set. We
+special-case `⋂₀ ∅ = ∅`. -/
+noncomputable def sInter (x : Set) : Set :=
+by { classical, exact dite x.nonempty (λ h, {y ∈ classical.some h | ∀ z ∈ x, y ∈ z}) (λ _, ∅) }
+
+prefix (name := Set.sInter) `⋂₀ `:110 := Set.sInter
+
+theorem mem_sInter {x y : Set} (h : x.nonempty) : y ∈ ⋂₀ x ↔ ∀ z ∈ x, y ∈ z :=
+begin
+  classical,
+  rw [sInter, dif_pos h],
+  simp only [mem_to_set, mem_sep, and_iff_right_iff_imp],
+  exact λ H, H _ (classical.some_spec h)
+end
+
+theorem to_set_sInter {x : Set.{u}} (h : x.nonempty) : (⋂₀ x).to_set = ⋂₀ (to_set '' x.to_set) :=
+by { ext, simp [mem_sInter h] }
```

Comment:
Both `pSet.to_set_sUnion` and `Set.to_set_sUnion` are simp.
```suggestion
@|simp] theorem to_set_sInter {x : Set.{u}} (h : x.nonempty) :
  (⋂₀ x).to_set = ⋂₀ (to_set '' x.to_set) :=
by { ext, simp [mem_sInter h] }
```

Reply:
This has the extra hypothesis `x.nonempty` though, and `simp` often doesn't work well in these cases.

Reply:
Wait, sorry. I meant
```suggestion
@[simp] theorem to_set_sInter (x : Set.{u}) : (⋂₀ x).to_set = ⋂₀ (to_set '' x.to_set) :=
by { obtain rfl | hs := s.eq_empty_or_nonempty, { simp },ext, simp [mem_sInter hs] }
```

Reply:
Does that actually work? I believe that in the empty case, the LHS should be empty while the RHS would be the universal set (currently afk so can't check).

Reply:
Yeah, this doesn't work. That first `simp` arrives as `Ø = set.univ`.